qcelemental/models/results.py

```diff
@@ -287,7 +288,7 @@ def _validate_derivs(cls, v, values, field):
         try:
             v = np.asarray(v).reshape(shape)
         except (ValueError, AttributeError):
-            raise ValueError(f"Derivative must be castable to shape {shape}!")
+            raise ValueError(f"Derivative must be castable to shape {shape}! len={np.asarray(v).size()}")
```

Comment:
Can this cause issues if `v` isn't something which can be reasonably cast through `asarray`?

Reply:
Yeah, info may not be worth the risk.

Reply:
If you want info, you can just post `v` so someone can see what the problematic entry was?